draft-ietf-ppm-dap.md

```diff
@@ -1554,12 +1562,17 @@ struct {
 where `prepare_resps` are the outbound prep steps computed in the previous step.
 The order MUST match `AggregationJobInitReq.prepare_inits`.
 
-The Helper responds to the Leader with HTTP status code 201 Created and a body
-consisting of the `AggregationJobResp`, with media type
-"application/dap-aggregation-job-resp".
+After receiving the response to its `AggregationJobInitReq`, the Leader makes
+an HTTP GET request to the aggregation job URI to check on the status of the
```

Comment:
is "aggregation job URI" well-defined? In any case, I think it would help the reader to either spell it out here or add a reference to the definition.

Reply:
It's not concretely defined--I copied the vernacular from the collection flow. I think I will expand it out instead of using the shorthand (and do the same for the other references).

Reply:
Done. I've replaced mention of `aggregation job URI` and `collection job URI` with the explicit paths.